tests/unit/readers/test_reader_postgres.py

```
@@ -383,9 +349,106 @@ def test_healthy_when_config_valid(self, reader: ReaderPostgres, pg_secret: dict
+        ok_conn = _make_mock_connection(_STATS_DESCRIPTION, [])
+
+        with (
+            patch("boto3.Session") as mock_session,
+            patch.object(pb, "psycopg2") as mock_pg,
+        ):
+            mock_client = MagicMock()
+            mock_client.get_secret_value.return_value = {"SecretString": json.dumps(pg_secret)}
+            mock_session.return_value.client.return_value = mock_client
+            mock_pg.connect.side_effect = [fail_conn, ok_conn]
+
+            rows, pagination = reader.read_stats(limit=10)
```

Ruff (0.15.10)

[warning] Unpacked variable `pagination` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)